client/ayon_core/pipeline/stagingdir.py

```diff
@@ -0,0 +1,217 @@
+from ayon_core.lib import Logger, filter_profiles, StringTemplate
+from ayon_core.settings import get_project_settings
+from ayon_core.pipeline.template_data import get_template_data
+
+from .anatomy import Anatomy
+from .tempdir import get_temp_dir
+
+STAGING_DIR_TEMPLATES = "staging"
```

Comment:
I would get rid of this constant, or move it elsewhere, e.g. to anatomy.

Reply:
This is removed.